converter/src/test/resources/converter/errantDefaultedNode.xml

```diff
@@ -15,7 +15,7 @@
             -->
             <component>
                 <section>
-                    <templateId root="867.5309"/>
+                    <templateId root="2.16.840.1.113883.10.20.27.1.2" extension="2017-07-01"/>
```

Comment:
Possibly a problem with this is that we already have the clinical document above.  So the Jenny decoder will be invoked twice.

Reply:
Hm... I put clinical document so that Jenny decoder would actually go and replace the template id. All the tested passed but you could be right that there is an edge case I missed.

Reply:
You could probably just replace it with the ACI section template ID (and update the test to use the ACI section as well).